addons/mail/static/src/web/chat_window/chat_window.js

```diff
@@ -154,4 +159,55 @@ export class ChatWindow extends Component {
         this.chatWindowService.close(this.props.chatWindow, options);
         this.chatWindowService.notifyState(this.props.chatWindow);
     }
+
+    get orderedActions() {
+        return this.actions.sort((act1, act2) => act1.sequence - act2.sequence);
+    }
+
+    get actions() {
+        const acts = [];
+        if (
+            this.thread?.allowCalls &&
+            this.thread !== this.rtc.state.channel &&
+            !this.props.chatWindow.hidden
```

Comment:
This condition `!this.props.chatWindow.hidden` is not checked for other buttons?

Reply:
`this.ui.isSmall` implicitly means chat window is definitely not hidden